### DISCUSSION TITLE
Can't connect to eXo server with Android App

### DISCUSSION THREAD

**Discussion by Arkronus** [exoplatform/exo-community#54] [Q&A]
Hi! I 
Can't connect to eXo server with [Android App](https://play.google.com/store/apps/details) and receive this error. What URL I should enter?

![изображение](https://github.com/exoplatform/exo-community/assets/6567321/c6a52813-95d3-4a20-accc-8824dcd70cec)

Sites works normally if I open it via mobile browser.

**Comment by rdenarie** [comment]
Hi @Arkronus 
With your desktop browser, you can access to parameters, and choose "Connect easily to eXo from your mobile device with this QR code."

It will display you a qr code, you can scan with eXo mobile app.

**Reply by Arkronus** [reply]
Hi @rdenarie 

I can't see this option. Here is my Settings screen

![exo-settings](https://github.com/exoplatform/exo-community/assets/6567321/73c68ccd-1a16-4ff7-a96e-d1793d0eaf31)

Using EXO 6.4 Community Edition

**Reply by rdenarie** [reply]
My bad, the qr code is in eXo 6.5

Which url do you put in your application ? 
In fact, it must an url, and cannot be an IP.
So you should put something like  : 
https://www.mytestdomain.com

**Reply by Arkronus** [reply]
I see, this must be the reason. We are connecting to eXo with IP address, but it works for iPhone app.

**Reply by Arkronus** [reply]
Small update. I have provided or Exo instance with domain but still was not able to connect with android mobile app. It turns out that it works only with https connection,
